users/templates/signup.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign Up</title>
+</head>
+
+<body>
+    <div class="container jumbotron">
+        {% if user.is_authenticated %}
+        <strong>You Are AllReady Signup with This Email {{ user.email }}.</strong><br>
```

Comment:
Spelling mistake `AllReady` -> `already`.

The message should be The user with this <email> is already exists.`